cleanlab_studio/cli/dataset/upload_helpers.py

```diff
@@ -3,6 +3,7 @@
 """
 import asyncio
 import decimal
+import os
```

Comment:
nit: we generally try to use pathlib rather than `os` and strings for our pathing